src/tests/bulkConcepts/containers/BulkConceptsPage.test.js

```diff
@@ -69,7 +69,7 @@ describe('Test suite for BulkConceptsPage component', () => {
     const wrapper = mount(<Router>
       <BulkConceptsPage {...props} />
     </Router>);
-    expect(wrapper.find('#emptyConcept').text()).toEqual('No concepts found');
+    expect(wrapper.find('.rt-noData').text()).toEqual('No concept!');
```

Comment:
Is no concept better than no concepts found?

Reply:
It makes sense. Updating it right away.

Reply:
done